addons/mail/models/fetchmail.py

```diff
@@ -223,8 +223,8 @@ def fetch_mail(self):
                         try:
                             imap_server.close()
                             imap_server.logout()
-                        except OSError:
-                            _logger.warning('Failed to properly finish imap connection: %s.', server.name, exc_info=True)
+                        except (OSError, IMAP4.error) as err:
+                            _logger.warning('Failed to properly finish imap connection: %s.', tools.ustr(err), exc_info=True)
```

Comment:
Do like this:
```
except (OSError, IMAP4.error):
     _logger.warning('Failed to properly finish imap connection: %s.', server.name, exc_info=True)
```

Also apply change here https://github.com/odoo/enterprise/blob/bba56b45da0d5926a7017d62d68164162e52dcb4/l10n_cl_edi/models/fetchmail_server.py#L112